.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: local
+    hooks:
+      - id: mojo-format
+        name: mojo-format
+        entry: mojo format -l 80
+        language: system
+        files: '\.(mojo|🔥)$'
+        stages: [commit]
+      - id: check-license
+        name: check-license
+        entry: mojo stdlib/scripts/check_license.mojo
```

Comment:
**Quibble** Perhaps rename to `check-licenses.mojo` (dash instead of underscore) for consistency in the other script names?  Don't feel strongly.

Reply:
Done